doc/mgr/dashboard.rst

```diff
@@ -249,6 +252,13 @@ The default value is 45 seconds.
 Enabling the Embedding of Grafana Dashboards
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note:: 
+  The embedding of Grafana dashboards into the Ceph Manager Dashboard is
+  currently work in progress. This section documents the backend configuration.
+  The corresponding changes to the WebUI have not been merged yet. You can
+  follow the development process in `PR#23666
+  <https://github.com/ceph/ceph/pull/23666>`_.
```

Comment:
I don't know if it makes sense to add these WIP things to the documentation. As soon as #23666 is merged, this note is outdated.

Reply:
> I don't know if it makes sense to add these WIP things to the documentation. As soon as #23666 is merged, this note is outdated.

@sebastian-philipp Agreed, it is somewhat confusing, hence my intention to add a clarifying note to this section.

It was added when the corresponding backend functionality was merged, with the intention of adding the missing frontend code that depends on it later. 

If #23666 obsoletes this part of the documentation, that PR should take care of removing or updating it accordingly.